README.md

```diff
@@ -25,6 +25,18 @@ Make sure you have GCC-4.6, G++-4.6, and GFortran-4.6 installed. For
 a full list of external dependencies on Ubuntu you can look at our
 [Travis configuration](https://github.com/graalvm/sulong/blob/master/.travis.yml).
 
+On the Mac you can use Homebrew:
+
+    brew install gcc
```

Comment:
Ok, I've removed the `README` stuff, so it's now just the library change. I think I almost have the right set of flags for Mac and Homebrew now, so I'll keep working on that and do a separate PR.